performant-software/dm-2
# Text editor features

### What this PR does

- Per #67:
  - Add "font family" selection to text editor
- Per #184:
  - Change font size dropdown to use pt units/numbers
- Per #360:
  - Add "blockquote" tool to text editor
- Per #364:
  - Add "horizontal rule" tool to text editor

### Status


### Steps to test

1. Visit https://dm-2-staging.herokuapp.com/
2. Log in
3. Open or create a project with Write or Admin access
4. Open and check out or create a text document

Issue #67:

5. Select some text, then use the font family dropdown in the toolbar to change the font from "sans-serif" to something else, and verify that its font changes successfully

Issue #184:

6. Select some text, then use the font size dropdown in the toolbar to change the size from 12 pt to something else, and verify that its size changes successfully

Issue #360:

7. Select some text, then use the "blockquote" button in the toolbar to create a blockquote, and verify that the block of text becomes indented with a faint gray line on the left side

Issue #364:

8. Use the "horizontal rule" button in the toolbar to create a horizontal rule, and verify that a gray horizontal line appears in the document
